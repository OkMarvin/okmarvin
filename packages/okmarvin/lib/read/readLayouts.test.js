const readLayouts = require('./readLayouts')
test('returns default value', async () => {
  const data = await readLayouts(__dirname, {})
  expect(data).toEqual({ layoutHash: [], layouts: Object.create(null) })
})
test('returns layouts from fixture', async () => {
  const data = await readLayouts(__dirname, {
    'index.js': ['index.js']
  })
  expect(data).toMatchSnapshot()
})
