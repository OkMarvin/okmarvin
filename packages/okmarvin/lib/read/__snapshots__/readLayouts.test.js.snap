// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns layouts from fixture 1`] = `
Object {
  "layoutHash": Array [
    "3ebce70467b166dda0d81ecb6e74679f",
  ],
  "layouts": Object {
    "index.js": [Function],
  },
}
`;
